.github/CONTRIBUTING.md

```diff
@@ -30,7 +32,39 @@ No pull request template is provided on GitHub. The expected changes are often a
 
 **Tip:** Make it small! A focused PR gives you the best chance of having it accepted. Then, repeat if you have more to propose!
 
-### How to prepare
+### Setup a development installation of Vagrant
+
+There are a few prerequisites for setting up a development environment with Vagrant. Ensure you have the following installed on your machine:
+
+* git
+* bsdtar
+
+#### Install Ruby
+
+It's nice to have a way to control what version of ruby is installed, so you may want to install [rvm](https://rvm.io/rvm/install), [chruby](https://github.com/postmodern/chruby#install) or [rbenv](https://github.com/rbenv/rbenv#installation). For Windows [ruby installer](https://rubyinstaller.org/) is recommended.
+
+#### Setup Vagrant
+Clone Vagrant's repository from GitHub into the directory where you keep code on your machine:
+
+```
+  $ git clone https://github.com/hashicorp/vagrant.git
+```
+
+Next, cd into that path. All commands will be run from this path:
```

Comment:
Next, move into the newly created `./vagrant directory:

```
$ cd ./vagrant
```

All commands will be run from this path. Now, run the `bundle` command to install the Ruby dependencies:

```
$ bundle install
```